assets/css/all.sass

```diff
@@ -30,6 +30,8 @@ html
 body
   height: 100%
   min-height: 100%
+  max-width: 1200px
+  margin: 0 auto
```

Comment:
Concept ACK. Testing it reminded me that this does not work as-is with the dark mode (see #328).

Reply:
Thanks! Should be fixed now.